src/tests/basic/test_arit.cpp

```diff
@@ -597,8 +598,9 @@ void test_pow()
     assert(std::abs(rcp_static_cast<const RealDouble>(r2)->i - 0.501187233627) < 1e-12);
 }
 
- void test_log()
- {
+void test_log()
+{
+    RCP<const Basic> x = symbol("x");
```

Comment:
In test files, symbols, prior to the `symbol` method, were created like this
`RCP<const Basic> x = rcp(new Symbol("x"));`
Is there any other reason for not doing it that way?